common/objectcachev2.h

```diff
@@ -120,37 +114,32 @@ class ObjectCacheV2 {
         SCOPED_LOCK(maplock);
         auto it = map.find(keyitem);
         if (it == map.end()) {
-            // item = new Box(std::forward<KeyType>(key));
-            auto rt = map.emplace(key);
+            auto rt = map.emplace(std::forward<KeyType>(key));
             if (rt.second) item = (Box*)&*rt.first;
-        } else
+        } else {
             item = (Box*)&*it;
+        }
         assert(item);
         item->timestamp = photon::now;
-        cycle_list.pop(item);
-        cycle_list.push_back(item);
+        lru_list.pop(item);
+        lru_list.push_back(item);
         return *item;
     }
 
     uint64_t __expire() {
-        intrusive_list<Box> delete_list;
         uint64_t now = photon::now;
```

Comment:
It's simpler to use now - lifespan

Reply:
What if `lifespan` has been set a extra-big number (like UINT64_MAX, present object in objcache never reclaim

sat_add is necessary.

Reply:
sat_sub(now, lifespan)